src/core/request.js

```diff
@@ -79,17 +79,21 @@ Request.prototype.request = function(servers, callback_) {
     // just in case
     this.emit = _.noop;
     this.cancel();
+    this.remote.removeListener('connected', doRequest);
   }, this._timeout);
 
-  function onResponse() {
-    clearTimeout(timeout);
-  }
-
   if (this.remote.isConnected()) {
     this.remote.on('connected', doRequest);
   }
 
-  this.once('response', onResponse);
+  function onRemoteError() {}
+  this.remote.once('error', onRemoteError); // e.g. rate-limiting slowDown error
```

Comment:
rippled has throttling, if you do too many requests too fast it will give you a slowDown message